batch/batch/cloud/gcp/driver/create_instance.py

```diff
@@ -43,7 +43,8 @@ def create_vm_config(
     _, cores = gcp_machine_type_to_worker_type_and_cores(machine_type)
 
     region = instance_config.region_for(zone)
-
+    accelerator_family = machine_type.split("-")[0]
+    docker_run_gpu_args = "--runtime=nvidia --gpus all" if machine_type_to_gpu(accelerator_family) else ""
```

Comment:
Can you use single quotes? Sorry for being nit picky. I think the entire rest of the code base is almost entirely single quotes.